utils/spock_create_subscriber/spock_create_subscriber.c

```
@@ -0,0 +1,1751 @@
+		num++;
+		c = strtok(NULL,",");
+	}
+	pfree(databases);
+	return result;
+}
+
+static char *
+generate_restore_point_name(void)
+{
+	char *rpn = malloc(NAMEDATALEN);
+	snprintf(rpn, NAMEDATALEN-1, "spock_create_subscriber_%lx", random());
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)